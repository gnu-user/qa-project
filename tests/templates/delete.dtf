02_seller__________SS_000000.00