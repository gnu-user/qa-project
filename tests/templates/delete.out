Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the username to delete:
Invalid transaction, you can not delete yourself.